src/Blocks/ChildDatabase.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Notion\Blocks;
+
+use Notion\Common\RichText;
+
+/**
+ * @psalm-import-type BlockJson from Block
+ *
+ * @psalm-type ChildDatabaseJson = array{
+ *      child_database: array{ title: string },
```

Comment:
Possibly not applicable, but it is worth documenting `title: non-empty-string` or similar, when some constraints are also enforced by `notion.com` on some of this stuff: prevents a lot of runtime waste of time and validation :)